filter_plugins/ceph-ansible.py

```diff
@@ -0,0 +1,13 @@
+from ansible import errors
+
+def only_one_true(a, *args):
+    params = list(args)
+    params.append(a)
+    it = iter(params)
+    return any(it) and not any(it)
```

Comment:
This will not always be false. try it out.

The first any(it) iterators through the listiterator (think generator) and stops when it finds a True. The next one iterators through the _remaining_ listiterator items looking for another True, if it finds it this function returns false, if it does not find it (there was only a single True) this function returns True

it = iter([False, True, False])
any(it) and not any(it)  # This returns True

it = iter([False, False, False])
any(it) and not any(it)  # This returns False

it = iter([False, True, True])
any(it) and not any(it)  # This returns False